examples/linux64/file-reading/reading_file.c

```diff
@@ -0,0 +1,73 @@
+#include <stdlib.h>
+#include <stdio.h>
+
+#define	O_RDONLY	0x0000		/* open for reading only */
+#define	O_WRONLY	0x0001		/* open for writing only */
+#define	O_RDWR		0x0002		/* open for reading and writing */
+#define	O_ACCMODE	0x0003		/* mask for above modes */
+
+inline size_t read(int fd, void *buf, size_t count){
+	return (size_t) syscall(0, fd, buf, count);
+}
+
+inline int open(const char *filename, int mode){
```

Comment:
Can those files be moved into the libc folder somewhere?

Reply:
I could make the corresponding *.h and *.c for these functions and place them inside.

Reply:
Sounds like a good idea!